<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotFrame2</class>
 <widget class="QFrame" name="PlotFrame2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QGroupBox" name="gbPlot">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>440</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbPlotControls">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>530</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Plot Controls</string>
   </property>
   <widget class="QPushButton" name="btnDetach">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detach</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbSpan">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="suffix">
     <string> s</string>
    </property>
    <property name="minimum">
     <number>120</number>
    </property>
    <property name="maximum">
     <number>3600</number>
    </property>
    <property name="singleStep">
     <number>60</number>
    </property>
    <property name="value">
     <number>600</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
